pandas/core/series.py

```diff
@@ -3039,13 +3039,57 @@ def notnull(self):
 
     def dropna(self, axis=0, inplace=False, **kwargs):
         """
-        Return Series without null values
+        Remove missing values.
+
+        Return Series without NA values.
+
+        Parameters
+        ----------
+        axis : {0 or 'index'}, default 0
+            There is only one axis to drop values from.
+        inplace : bool, default False
+            If True, do operation inplace and return None.
+        **kwargs
```

Comment:
so we should just remove kwargs entirely (and need to remove the check in the code as well). This was originally fro compat with DataFrame but I think its ok (its possible that we are testing this though, so may need to modify a test). Could do this in a followup. cc @jorisvandenbossche

Reply:
So should I delete the **kwargs from the docstring already, or later when the code is fixed? And can I do the follow-up PR on this issue in the code base, or will you do it when the testing is clarified?

Reply:
Let's do this in a follow-up, and keep this PR purely docstring. So @donK23 you can leave it as is for now. 
And certainly welcome to do such a follow-up PR